### PR TITLE
Jomefavourite/issuenew #833

### PR DESCRIPTION
**Pull request by jomefavourite**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by jomefavourite** [comment]
@Tanuj22

File: package-lock.json [hunk 1]
```diff
@@ -5,9 +5,19 @@
   "requires": true,
   "dependencies": {
     "@ardatan/aggregate-error": {
-      "version": "0.0.1",
-      "resolved": "https://registry.npmjs.org/@ardatan/aggregate-error/-/aggregate-error-0.0.1.tgz",
-      "integrity": "sha512-UQ9BequOTIavs0pTHLMwQwKQF8tTV1oezY/H2O9chA+JNPFZSua55xpU5dPSjAU9/jLJ1VwU+HJuTVN8u7S6Fg=="
+      "version": "0.0.6",
```
Comment (leecalcote) [inline package-lock.json:8]: This seems like a very significant change. Is this necessary?

Comment (Jashpatel1) [reply package-lock.json:8]: I think this change is caused due to downgrading the version of Gatsby.

Comment (Tanuj22) [reply package-lock.json:8]: Everything looks good @jomefavourite . You just don't have to commit the changes to this file and we are good to go

Comment (jomefavourite) [reply package-lock.json:8]: @Tanuj22 do you mean I should revert the changes?

Comment (Tanuj22) [reply package-lock.json:8]: Yes

**Comment by jomefavourite** [comment]
Changed the version back @Tanuj22

**Comment by Tanuj22** [comment]
@jomefavourite there are still changes in package.lock.json. There shouldn't be any changes

**Comment by leecalcote** [comment]
Hi @jomefavourite it looks like your remote branch might be gone. What would you like to do with this PR?

**Comment by jomefavourite** [comment]
I want to start all over and make it work once and for all. So I deleted the previous fork and forked the repo again.

**Comment by jomefavourite** [comment]
I'll be making a new PR soon

**Comment by Tanuj22** [comment]
Fixed in #952

**Pull request closed without merging**